src/Environment.swift

```diff
@@ -148,3 +171,17 @@ public enum Env {
         return ProcessInfo.processInfo.environment[key]
     }
 }
+
+private extension NSColor {
+    convenience init?(withRGBA rgba: String) {
+        let numberRegex = try! NSRegularExpression(pattern: "(\\d+\\.?\\d+)")
+        let nsrgba = rgba as NSString
+        let matches = numberRegex.matches(in: rgba, range: NSRange(location: 0, length: nsrgba.length))
+        let results = matches
+            .map { nsrgba.substring(with: $0.range) }
+            .flatMap { Double($0) }
+            .map { CGFloat($0) }
+        guard results.count == 4 else { return nil }
+        self.init(red: results[0], green: results[1], blue: results[2], alpha: results[3])
+    }
```

Comment:
Cleaning up? No, that's not possible. But I have an idea how to avoid the NSString bridging:

```Swift
let numberRegex = try! NSRegularExpression(pattern: "(\\d+\\.?\\d+)")
let matches = numberRegex.matches(in: rgba, range: NSRange(location: 0, length: rgba.count))
let results = matches
                .map {
                    let startIndex = rgba.index(rgba.startIndex, offsetBy: $0.range.location)
                    let endIndex = rgba.index(startIndex, offsetBy: $0.range.length)
                    return rgba.substring(with: startIndex..<endIndex)
                }
                .flatMap { Double($0).map({ CGFloat($0) }) }
```